docker/influxdb/setup-influxdb.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+apk add curl \
```

Comment:
Sorry, I missed that: does this script run on each container launch and thus trying to install curl? I think it might be undesired. If curl is not installed, maybe you can just loop  `influx -execute...` until success (or timeout)?

Reply:
Ah, I missed that as well.

I don't think we can rely on any external network calls in a running container, as we may be in a darksite.

Reply:
Yeah I should be able to use `influx -execute`. I'll update.